### ISSUE TITLE
Couldn't compile the website (Rust 1.39+)

### ISSUE THREAD

**Issue by eisterman** [vulkano-rs/vulkano-www#71]
I've tried to compile in my system but the result is every time a compile error.

With 1.39-0-stable toolchain:
```$ ADDR=0.0.0.0:8000 cargo run
   Compiling vulkano-www v0.1.0 (/home/fpasqua/git/vulkano-www)
error[E0283]: type annotations required: cannot resolve `std::string::String: std::convert::AsRef<_>`
   --> src/lib.rs:153:45
    |
153 |                 .insert_str("body", e.key().as_ref())
    |                                             ^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0283`.
error: could not compile `vulkano-www`.
```

With 1.41.0-nightly (a605441e0 2019-12-15) toolchain:
```
$ ADDR=0.0.0.0:8000 cargo run
   Compiling vulkano-www v0.1.0 (/home/fpasqua/git/vulkano-www)
error[E0283]: type annotations needed for `std::string::String`
   --> src/lib.rs:153:45
    |
146 |     let body = body.into();
    |         ---- consider giving `body` a type
...
153 |                 .insert_str("body", e.key().as_ref())
    |                                             ^^^^^^ cannot infer type for struct `std::string::String`
    |
    = note: cannot resolve `std::string::String: std::convert::AsRef<_>`

error[E0283]: type annotations needed for `std::string::String`
   --> src/lib.rs:185:45
    |
178 |     let body = body.into();
    |         ---- consider giving `body` a type
...
185 |                 .insert_str("body", e.key().as_ref())
    |                                             ^^^^^^ cannot infer type for struct `std::string::String`
    |
    = note: cannot resolve `std::string::String: std::convert::AsRef<_>`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0283`.
error: could not compile `vulkano-www`.
```

Now I will try to fix this for my computer adding some type annotation, but I don't know perfectly how the deployment is done, because is really strange that it works on the deploy and not with the last versions of rust.

**Comment by eisterman** [comment]
Changing 'as_ref()' with 'as_str()' fix the problem for Rust-1.39 and Rust-Nightly.

**Comment by AustinJ235** [comment]
I believe this is solved now.
